server/logging/pubsub.go

```diff
@@ -29,14 +35,90 @@ func NewPubSubLogWriter(projectId string, topicName string, logger log.Logger) (
 		"msg", "GCP PubSub writer configured",
 		"project", projectId,
 		"topic", topicName,
+		"include_attributes", includeAttributes,
+		"decoration_attributes", decorationAttributes,
 	)
 
 	return &pubSubLogWriter{
-		topic:  topic,
-		logger: logger,
+		topic:                topic,
+		logger:               logger,
+		includeAttributes:    strings.Split(includeAttributes, ","),
+		decorationAttributes: strings.Split(decorationAttributes, ","),
 	}, nil
 }
 
+func asString(value interface{}) (string, bool) {
```

Comment:
Could it be reasonable to replace both uses of `asString` with `fmt.Sprint`? That way it would simplify error handling and also handle a larger range of types.

Reply:
I toyed with the idea of just using the json.RawValue, but I think we want strings to be unquoted, and probably just omit nulls.  I don't think we want to commit to the Go representation of compound types?

I don't think compound types are useful at all in this context, and the scalar types when decoding a json to `interface{}` are `string`, `boolean`, `float64` and `null`.

Maybe I should filter for just scalar type and use fmt.Sprint?

Reply:
Fair enough.